tests/console/rust.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
```

Comment:
Please just remove the year

```suggestion
# Copyright SUSE LLC
```

Reply:
Can i ask why? We use the date everywhere so i am not sure why this is an exception

Reply:
That's not an exception but the rule for *new* files, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style . Only for existing files it should be kept as is. I assume other reviewers do not look out for that in all cases.